### PR TITLE
Fix controller target

### PR DESCRIPTION
**Pull request by dzhengfy**
Change the usage according to avocado-vt PR 2685.

Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

**Comment by dzhengfy** [comment]
Depends on avocado-vt PR 2685

File: libvirt/tests/src/controller/controller_functional.py [hunk 1]
```diff
@@ -763,7 +763,8 @@ def add_device_xml(vm_xml, device_type, device_cfg_dict):
                     if 'model' in one_contrl:
                         contr_dict.update({'controller_model': one_contrl['model']})
                     if 'busNr' in one_contrl:
-                        contr_dict.update({'controller_busNr': one_contrl['busNr']})
+                        cntl_target = "{'busNr': %s}" % one_contrl['busNr']
```
Comment (chunfuwen) [inline libvirt/tests/src/controller/controller_functional.py:766]: is it native that  cntl_target is represented as dict ,rather one string?

Comment (dzhengfy) [reply libvirt/tests/src/controller/controller_functional.py:766]: @chunfuwen , this is changed according to the usage of avocado-framework/avocado-vt#2685.

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**